.travis.yml

```diff
@@ -10,16 +9,37 @@ php:
   - hhvm
  
 matrix:
-    allow_failures:
-        - php: hhvm
+  fast_finish: true
+  include:
+    - php: 5.3
+      env: SYMFONY_VERSION='2.3.*' FOSUSERBUNDLE_VERSION='~1.3'
+    - php: 5.3
+      env: SYMFONY_VERSION='2.7.*' FOSUSERBUNDLE_VERSION='2.0.*@dev'
+    - php: 5.6
+      env: SYMFONY_VERSION='2.3.*' FOSUSERBUNDLE_VERSION='~1.3'
+    - php: 5.6
+      env: SYMFONY_VERSION='2.7.*' FOSUSERBUNDLE_VERSION='2.0.*@dev'
+    - php: 5.6
+      env: SYMFONY_VERSION=2.8.*@dev FOSUSERBUNDLE_VERSION='2.0.*@dev'
+    - php: 5.6
+      env: SYMFONY_VERSION="3.0.x-dev as 2.8" FOSUSERBUNDLE_VERSION='2.0.*@dev'
+  allow_failures:
+    - php: hhvm
+    - env: SYMFONY_VERSION=2.8.*@dev
+    - env: SYMFONY_VERSION="3.0.x-dev as 2.8"
 
-env:
-  - SYMFONY_VERSION='2.3.*' FOSUSERBUNDLE_VERSION='~1.2'
-  - SYMFONY_VERSION='2.4.*' FOSUSERBUNDLE_VERSION='~1.2'
-  - SYMFONY_VERSION='2.3.*' FOSUSERBUNDLE_VERSION='2.0.*@dev'
-  - SYMFONY_VERSION='2.4.*' FOSUSERBUNDLE_VERSION='2.0.*@dev'
```

Comment:
previously, all builds were running with FOSUserBundle installed. Now, you have lots of jobs running without it. You could limit the number of jobs needed by choosing a FOSUB version by default (in ``env: global: ...`` to apply it to all jobs) and then just overwrite it for some jobs